python-avd/pyavd/_eos_designs/schema/schema_fragments/isis_system_id_format.schema.yml

```diff
@@ -20,4 +19,4 @@ keys:
     valid_values:
       - node_id
       - underlay_loopback
-    default: "node_id"
+    default: underlay_loopback
```

Comment:
you need to update the description as well

Reply:
Since you changed the default to underlay_loopback so please check if we have a test for node_id or not. If not then please add it

Reply:
We have test for node_id.

Updated key description.